catlin/pkg/validator/testdata.go

```diff
@@ -0,0 +1,83 @@
+package validator
+
+var (
+	validTask = `
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: valid
+  labels:
+    app.kubernetes.io/version: a,b,c
+  annotations:
+    tekton.dev/tags: a,b,c
+    tekton.dev/pipelines.minVersion: "0.12"
+    tekton.dev/displayName: My Example Task
+spec:
+  description: |-
+    A summary of the resource
+
+    A para about this valid task
+
+  steps:
+  - name: hello
+    image: abc.io/ubuntu:1.0
+    command: [sleep, infinity]
+  - name: foo-bar
+    image: abc.io/fedora:1.0@sha256:deadb33fdeadb33fdeadb33fdeadb33fdeadb33fdeadb33fdeadb33fdeadb33f
+`
+
+	taskWithInvalidImageRef = `
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: valid
+  labels:
+    app.kubernetes.io/version: a,b,c
+  annotations:
+    tekton.dev/tags: a,b,c
+    tekton.dev/pipelines.minVersion: "0.12"
+    tekton.dev/displayName: My Example Task
+spec:
+  description: |-
+    A summary of the resource
+
+    A para about this valid task
+
+  steps:
+  - name: hello
+    image: ubuntu
```

Comment:
@vdemeester @chmouel is this valid? shouldn't we insist on a tag ? 
cc: @SM43

Reply:
this will fail validation :D 
I went by the name of the task - `valid`

Reply:
This would be invalid indeed.